static/templates/ranking/ranking.html

```diff
@@ -0,0 +1,87 @@
+        </div>
+        <md-divider></md-divider>
+        <div>
+            <div ng-repeat="item in ranking.pendingRankings">
+
+                <div>
+                    <div class="md-data-table-row" layout="row" layout-align="start center"
+                         >
+                        <div class="md-data-table-row-cell cell-text" flex="40">{{item.worker_alias}}</div>
+                        <div class="md-data-table-row-cell cell-text" flex="40">{{item.module_name}}</div>
+                        <div class="md-data-table-row-cell cell-text" flex>
+                            <div checkrating click-handler="ranking.handleRatingSubmit" model="item" selected="item.current_rating"></div>
+                        </div>
+
+                    </div>
+                    <md-divider></md-divider>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div ng-show="!ranking.pendingRankings.length" class="animate-show">
+        <div class=" md-data-table-footer md-data-table-row md-data-table-row-cell cell-text" layout="row" layout-align="start center">No worker reviews found!</div>
+        <md-divider></md-divider>
+    </div>
+
+</md-data-table>
+
+<md-data-table class="table">
+    <div class="md-data-table-card-header" layout="row" layout-align="start center">
+        <div class="md-data-table-card-header-title" flex="35">Rank requesters</div>
+    </div>
+    <div class="table-content">
+        <div class="md-data-table-header" layout="row" layout-align="start center">
+            <div class="md-data-table-header-column" flex="25" layout="row" layout-alignment="start center">
+                Requester
+            </div>
+            <div class="md-data-table-header-column" flex="25" layout="row" layout-alignment="start center">
+                Project name
+            </div>
+            <div class="md-data-table-header-column" flex="25" layout="row" layout-alignment="start center">
+                Work completed
+            </div>
+            <div class="md-data-table-header-column" flex layout="row" layout-alignment="start center">
+                Rating
+            </div>
+        </div>
+        <md-divider></md-divider>
+        <div>
+            <div ng-repeat="item in ranking.requesterRankings">
+
+                <div>
+                    <div class="md-data-table-row" layout="row" layout-align="start center"
+                         >
+                        <div class="md-data-table-row-cell cell-text" flex="25">{{item.project.owner.alias}}</div>
+                        <div class="md-data-table-row-cell cell-text" flex="25">{{item.project.name}}</div>
+                        <div class="md-data-table-row-cell cell-text" flex="25">{{item.module_data.name}}</div>
+                        <div class="md-data-table-row-cell cell-text" flex>
+                            <div checkrating click-handler="ranking.handleRatingSubmit" model="item" selected="item.current_rating"></div>
```

Comment:
can you move actions to the top rather than per row (let's follow the specs), thanks